policy/lib/tekton/pipeline.rego

```diff
@@ -33,14 +32,14 @@ pipeline_label_selector(pipeline) := value if {
 
 	# Labels of the build Task from the SLSA Provenance v1.0 of a PipelineRun
 	values := [l | some build_task in build_tasks(pipeline); l := build_task.metadata.labels[task_label]]
-	count(lib.to_set(values)) == 1
+	count(data.lib.to_set(values)) == 1
```

Comment:
I don't understand not needing the import here, but in other places we do?

Reply:
Yeah, I'm not 100% sure either about this one either. I guess you can always reference things by their full path starting from data..?

Reply:
The root cause of the lint error is that `data.lib.tkn` is inside `data.lib` so importing `data.lib` there in the `data.lib.tkn` package is circular. But what if we still need to reference `data.lib` so we can do `lib.to_set`. I'm not sure. Maybe I'll add a todo.

Reply:
I see. Our package naming under `lib` is a bit messy. Maybe that package should be `lib.set_helpers`? Not something that needs to be fixed here, but I can file a story if you agree.

Reply:
Yeah that would be reasonable.